### PR TITLE
Arm presets

### PR DESCRIPTION
**Pull request by End10**

### CODE REVIEW

File: src/main/java/frc/robot/commands/OneBar/PID.java
Comment (kv-gh) [inline src/main/java/frc/robot/commands/OneBar/PID.java:19]: If you want to use the same name in the parameter as the field in the class, you can use the `this` keyword.

```java
public PID(OnebarSubsystem onebarsubsystem, double setPoint) {
  this.onebarsubsystem = onebarsubsystem;  
  this.setPoint = setPoint;
  ...
```

**Review by kv-gh — COMMENTED** [review]
Looks fine to me; going to wait on Dan for approval though.

File: src/main/java/frc/robot/commands/OneBar/ArmToHigh.java
Comment (mathyourlife) [inline src/main/java/frc/robot/commands/OneBar/ArmToHigh.java:15]: Does this need to be a sequential command group? The two commands are using two different subsystems. How about we use ParallelCommandGroup instead?

(Same comment for other commands)

**Review by mastermind-mayhem — APPROVED** [review]

**Review by awesomeatpiano — APPROVED** [review]

**Pull request merged**
